supabase/migrations/20260826073804_dedicated_native_builder.sql

```
@@ -0,0 +1,155 @@
+ALTER TABLE "public"."build_requests"
+  DROP CONSTRAINT IF EXISTS "build_requests_builder_pool_check";
+
+-- NOT VALID: avoid a blocking ACCESS EXCLUSIVE full scan of
+-- build_requests on apply. New/updated rows are still checked;
+-- validate later in a maintenance window if desired.
+ALTER TABLE "public"."build_requests"
+  ADD CONSTRAINT "build_requests_builder_pool_check"
+  CHECK (
+    (
+      "builder_pool" IS NULL
+      OR "builder_pool" = ANY (
```

SQLFluff (4.3.0)

[error] Function name not immediately followed by parenthesis.

(LT06)